src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Settings/ContentSettings.cs

```diff
@@ -0,0 +1,6 @@
+namespace OrchardCore.ContentManagement.Metadata.Settings;
+
+public class ContentSettings
```

Comment:
Can't we use the existing settings? Like the one where we have `Attachable` for parts, maybe types also have some already.

I am not a fan of `CodeManaged`. Suggestions: 
- IsSystemPart, IsSystemType
- IsRemovablePart, IsRemovableType

Reply:
I like `IsSystemType`. I did not want to use the existing settings like `ContentTypeSettings` because that one is managed by the UI.

Reply:
I changed it to `IsSystemType`